data/harbour-pure-maps.desktop

```diff
@@ -4,3 +4,6 @@ Exec=harbour-pure-maps
 Icon=harbour-pure-maps
 Type=Application
 X-Nemo-Application-Type=silica-qt5
+
+[X-HarbourBackup]
+BackupPathList=/home/nemo/.config/harbour-pure-maps
```

Comment:
Hi. I believe that path should be relative (from home) directory. Because in newer SFOS it is possible to have multiple users and default user is called "defaultuser" after fresh install.

Reply:
My bad

Reply:
@Karry thanks!